### DISCUSSION TITLE
VERSION.txt : install or not

### DISCUSSION THREAD

**Discussion by vermaete** [OpenDDS/OpenDDS#4647] [Q&A] [answered]
Hi,

When working on the cmake version of the meta-opendds is see the 'VERSION.txt' file is installed.

```
ERROR: opendds-3.28.1-r0 do_package: QA Issue: opendds: Files/directories were installed but not shipped in any package:
  /usr/share/dds
  /usr/share/dds/VERSION.txt
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
opendds: 2 installed and not shipped files. [installed-vs-shipped]
```

I don't really see a point of installing this file on a target?
Would it not be better be removed from the 'install' target

The content:
```
This is OpenDDS version 3.29.0-dev (NOT A RELEASE)

This software is open source and free of licensing fees.  See the
LICENSE file (in this directory) for license details.

For information about OpenDDS support see the README.md file.

For more information about OpenDDS please visit www.opendds.org.
```

br

**Comment by iguessthislldo** [comment]
That file is used [by the CMake package to get the version of OpenDDS](https://github.com/OpenDDS/OpenDDS/blob/master/cmake/opendds_version.cmake).

**Reply by vermaete** [reply]
Hi,
But no need in have in installed on a target?  What is now the case.

**Reply by iguessthislldo** [reply, marked as answer]
Oh, on a target? It might be nice to install, but no, it's not needed.
